subprojects/common/report-viewer-test-fixtures/src/main/kotlin/com/avito/report/MockReportsExtension.kt

```diff
@@ -1,13 +1,9 @@
 package com.avito.report
 
+import com.avito.logger.Logger
 import com.avito.test.http.MockDispatcher
 import okhttp3.mockwebserver.MockWebServer
-import com.avito.logger.Logger
-import org.junit.jupiter.api.extension.AfterEachCallback
-import org.junit.jupiter.api.extension.BeforeEachCallback
-import org.junit.jupiter.api.extension.ExtensionContext
-import org.junit.jupiter.api.extension.ParameterContext
-import org.junit.jupiter.api.extension.ParameterResolver
+import org.junit.jupiter.api.extension.*
```

Comment:
Are Import settings broken?

Reply:
We don't know how to make it in editorconfig.
These settings are in [IDE preferences](https://avito-tech.github.io/avito-android/docs/contributing/codestyle/#%D0%B5%D1%81%D0%BB%D0%B8-%D0%B0%D0%B2%D1%82%D0%BE%D0%BC%D0%B0%D1%82%D0%B8%D0%BA%D0%B0-%D1%81%D0%BB%D0%BE%D0%BC%D0%B0%D0%BB%D0%B0%D1%81%D1%8C)